cluster/addons/rbac/cluster-loadbalancing/glbc/clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:glbc
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups: [""]
+  resources: ["secrets", "endpoints", "services", "pods", "nodes", "namespaces", "configmaps", "events"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
+- apiGroups: ["extensions"]
```

Comment:
add `network.k8s.io` anywhere we refer to ingress resources

Reply:
done

Reply:
@liggitt Just so I'm clear, using networking.k8s.io is assuming that Ingress-GCE starts understanding that resource the same time these changes drop. Is that correct?

Reply:
we know ingress resources are addressable via both API groups... adding both here now saves us having to revisit these later when we switch ingress-gce to use the new API group. we're still granting the "extensions" API group as well, so it doesn't require ingress-gce to switch immediately